cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -329,6 +341,69 @@ var _ = Describe("dpdk", func() {
 		})
 	})
 
+	Context("dpdk application on different vendors", func() {
+		var nodeNames []string
+		var sriovInfos *sriovcluster.EnabledNodes
+		var err error
+		var out string
+
+		execute.BeforeAll(func() {
+			sriovInfos, err = sriovcluster.DiscoverSriov(sriovclient, namespaces.SRIOVOperator)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(sriovInfos).ToNot(BeNil())
+
+			nodeNames, err = nodes.MatchingOptionalSelectorByName(sriovInfos.Nodes)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(nodeNames)).To(BeNumerically(">", 0))
+		})
+
+		BeforeEach(func() {
+			if discovery.Enabled() {
+				Skip("Split VF test disabled for discovery mode")
+			}
+
+			CleanSriov()
+		})
+
+
+		DescribeTable("Test connectivity using the requested nic", func(vendorID, deviceID string) {
+			By("searching for the requested network card")
+			node, sriovDevice,exist := findSriovDeviceForDPDK(sriovInfos,nodeNames,vendorID, deviceID)
```

Comment:
nit: missing some spaces in the arg list here

Reply:
done